.github/workflows/go.yml

```diff
@@ -32,7 +32,11 @@ jobs:
           [ -z "$(git diff -- go.{mod,sum})" ] # Check there are no changes!
 
       - name: Test (main module)
-        run: go test -race ./...
+        run: go test -v -race ./...
+        env:
+          CLARIFY_USERNAME: ${{ secrets.CLARIFY_USERNAME }}
+          CLARIFY_PASSWORD: ${{ secrets.CLARIFY_PASSWORD }}
+          CLARIFY_ENDPOINT: ${{ secrets.CLARIFY_ENDPOINT }}
 
```

Comment:
I ended up setting the secrets in _environments_; so we should look into running from environments when running these tests. Maybe a separate step for the integration tests?

Probably we can skip the `-v` flag for both tasks (easier to find failure).

```go
  - name: Test (integration tests)
        run: go test ./test/...
        env:
          CLARIFY_USERNAME: ${{ secrets.CLARIFY_USERNAME }}
          CLARIFY_PASSWORD: ${{ secrets.CLARIFY_PASSWORD }}
          CLARIFY_ENDPOINT: ${{ secrets.CLARIFY_ENDPOINT }}
```

Other tests will not get `CLARIFY_USERNAME` etc set, and will skip the integration tests.

Reply:
Secrets where added to  two _environments_; so we should check out how selection of that works when we run scheduled tests.

Reply:
Maybe we need to do integration tests in a _separate_ job in order to pass in envrionement.... Looks harder than I expected.